doc/conf.py

```diff
@@ -42,6 +42,11 @@
     print "ipython: %s, %s" % (IPython.__version__, IPython.__file__)
 except ImportError:
     print "no ipython"
+try:
+    import seaborn
```

Comment:
Docs look good on the server- http://xray.readthedocs.org/en/latest/plotting.html#discrete-colormaps, but for some reason this line causes my local build to to hang on `reading sources... [ 98%] plotting`. 

Also the first few plots show up in a window during the build, like in matplotlib interactive mode. Not sure why this happens- maybe caused by some config files somewhere on my machine.